Sources/Sentry/Profiling/SentryLaunchProfiling.m

```diff
@@ -307,6 +307,8 @@
     sentry_launchTracer = [[SentryTracer alloc] initWithTransactionContext:context
                                                                        hub:nil
                                                              configuration:config];
+
+    removeAppLaunchProfilingConfigFile();
```

Comment:
`m: I think it would be great to add the major points of the PR description and explain here as a code comment why we call `removeAppLaunchProfilingConfigFile`.